.swift-version

```diff
@@ -0,0 +1 @@
+3.0-PREVIEW-4
```

Comment:
Preview 5 is not released yet (this is mainly for Linux support preparation): https://swift.org/download/#previews 😅